### ISSUE TITLE
Template for new spoons is missing

### ISSUE THREAD

**Issue by hjklapp** [Hammerspoon/hammerspoon#3188]
When I use `hs.spoon.newSpoon` I get a file-not-found error because the file is not included with hammerspoon.

```
...merspoon.app/Contents/Resources/extensions/hs/spoons.lua:34:
/Applications/Hammerspoon.app/Contents/Resources/extensions/hs/templates/init.tpl: No such file or directory
```

I just downloaded the file in question from here
https://github.com/Hammerspoon/hammerspoon/blob/master/extensions/spoons/templates/init.tpl

But still seems like it should work out of the box.

BTW: I have to disable the ReloadConfiguration.spoon, otherwise hammerspoon is reloaded before the template can be instantiated. No big deal, but would be nice if it was mentioned in the docs for `hs.spoon.newSpoon`.

**Comment by latenitefilms** [comment]
Looks like this file wasn't thought about when @cmsj changed how Hammerspoon is organised internally?

In the meantime as a workaround you could manually copy the file across?

**Comment by hjklapp** [comment]
Yes, I downloaded the file from GitHub and stored it in a convenient location. 
`hs.spoons.newSpoon` has a parameter for the template file path, so there is a workaround.

**Comment by danielo515** [comment]
Just wanted to report that this is still present

**Comment by yoggiland** [comment]
Template init.tpl is still missing, and I only managed to solve it do to this issue ticket entry here.
